tests/test_session.py

```diff
@@ -135,27 +135,42 @@ def test_open_image(mocker, session, args, kwargs, expected_args, expected_kwarg
 
 
 @pytest.mark.parametrize("args,kwargs,expected_args,expected_kwargs", [
-    # Open plain image
+    # Open complex image with default (AMPLITUDE) setting
     (["subdir/image.fits"], {},
-     ["subdir/image.fits", "", True, False, AE.AMPLITUDE], {"make_active": True, "update_directory": False}),
-    # Open plain image; select different HDU
-    (["subdir/image.fits"], {"hdu": "3"},
-     ["subdir/image.fits", "3", True, False, AE.AMPLITUDE], {"make_active": True, "update_directory": False}),
-    # Open complex image
-    (["subdir/image.fits"], {"complex": True},
-     ["subdir/image.fits", "", True, True, AE.AMPLITUDE], {"make_active": True, "update_directory": False}),
-    # Open plain image; update file browser directory
-    (["subdir/image.fits"], {"update_directory": True},
-     ["subdir/image.fits", "", True, False, AE.AMPLITUDE], {"make_active": True, "update_directory": True}),
-    # Open plain image; don't make active
-    (["subdir/image.fits"], {"make_active": False},
-     ["subdir/image.fits", "", True, False, AE.AMPLITUDE], {"make_active": False, "update_directory": False}),
+     ["subdir", 'AMPLITUDE("image.fits")', "", False, True], {"make_active": True, "update_directory": False}),
+    # Open complex image (PHASE)
+    (["subdir/image.fits"], {"expression": CC.PHASE},
+     ["subdir", 'PHASE("image.fits")', "", False, True], {"make_active": True, "update_directory": False}),
+    # Append complex image (REAL)
+    (["subdir/image.fits"], {"expression": CC.REAL, "append": True},
+     ["subdir", 'REAL("image.fits")', "", True, True], {"make_active": True, "update_directory": False}),
+    # Open complex image (IMAG); update file browser directory
+    (["subdir/image.fits"], {"expression": CC.IMAG, "update_directory": True},
+     ["subdir", 'IMAG("image.fits")', "", False, True], {"make_active": True, "update_directory": True}),
+])
+def test_open_complex_image(mocker, session, args, kwargs, expected_args, expected_kwargs):
```

Comment:
This test is now failing -- you need to update the name of the keyword parameter in the test cases.